s3-uploader/tkn/task-oras.yaml

```
@@ -0,0 +1,90 @@
+        aws_access_key_id     = $(cat /opt/aws-credentials/access-key)
+        aws_secret_access_key = $(cat /opt/aws-credentials/secret-key)
+        EOF
+        cat <<EOF > /home/1001/.aws/config
+        [default]
+        region = $(cat /opt/aws-credentials/region)
+        EOF
+
+        region = $(cat /opt/aws-credentials/region)
+
+        aws s3 cp --recursive /opt/results/$(params.qe-workspace-subpath)/ s3://$(params.s3-bucket)/$(params.s3-path) --acl public-read
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)